src/amo/components/Category/index.js

```diff
@@ -24,7 +18,10 @@ export class CategoryBase extends React.Component {
     category: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
     errorHandler: PropTypes.object.isRequired,
+    filters: PropTypes.object,
     loading: PropTypes.boolean,
+    paginationQueryParams: PropTypes.object,
```

Comment:
it's used here, no? https://github.com/mozilla/addons-frontend/blob/395c53e29af341a7508ca4e04b387cabe80a37bc/src/amo/components/Category/index.js#L68